lib/src/messages/util.dart

```diff
@@ -95,3 +147,150 @@ int validateSessionIDOrRaise(Object? sessionID, String errorMsg, [String? field]
 
   return sessionID;
 }
+
+String? validateInt(Object value, int index, String message) {
+  if (value is! int) {
+    return invalidDataTypeError(
+      message: message,
+      index: index,
+      expectedType: int,
+      actualType: value.runtimeType.toString(),
+    );
+  }
+  return null;
+}
+
+const minID = 1;
+const maxID = 9007199254740992;
```

Comment:
```suggestion
const maxID = 1<<53;
```
also move to the top of the file